space-cli/plugins.go

```diff
@@ -0,0 +1,31 @@
+package main
+
+import (
+	"fmt"
+	"os"
+	"plugin"
+
+	"github.com/spf13/cobra"
+)
+
+func getplugin() (*cobra.Command, error) {
+	mod := fmt.Sprintf("%s/cmd_%s.so", getSpaceCliDirectory(), latestVersion)
+	plug, err := plugin.Open(mod)
+	if err != nil {
+		fmt.Println(err)
+		os.Exit(1)
+	}
+	_, err = plug.Lookup("")
```

Comment:
You need to look up the Commands function here. It will be of type `func() *cobra.Command`